### ISSUE TITLE
[rqt_controller_manager] Don't assume controller_manager name

### ISSUE THREAD

**Issue by adolfo-rt** [ros-controls/ros_control#166]
The `rqt_controller_manager` plugin assumes that a controller manager's ROS API lives in a namespace ending in `controller_manager`, for instance

```
/controller_manager
/my_robot/controller_manager
```

There are scenarios in which it would make sense to remap `controller_manager` to something else, for instance in robots that have more than one controller manager.

I would consider a controller manager namespace any path containing the six services that define its ROS API. The helpers in #165 implement this idea.

@kphawkins, do you see any inconvenience in modifying how the controller manager namespace is handled and reported in the `rqt_controller_manager` GUI?.

**Comment by kphawkins** [comment]
@adolfo-rt Not at all. In fact I do it myself here: https://github.com/gt-ros-pkg/universal_robot/blob/hydro-devel-c-api/ur_py_utils/src/ur_py_utils/ur_controller_manager.py
